strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/domain/ArtifactGroup.java

```diff
@@ -0,0 +1,37 @@
+package org.carlspring.strongbox.domain;
+
+import org.carlspring.strongbox.data.domain.GenericEntity;
+
+import javax.annotation.Nonnull;
+import javax.persistence.MappedSuperclass;
+import javax.persistence.OneToMany;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+@MappedSuperclass
+public abstract class ArtifactGroup
```

Comment:
I don't need it, see [here](https://github.com/strongbox/strongbox-db/pull/5/files)

Reply:
there is `name` property for `ArtifactGroup` class there.
anyway, I think we should have `ArtifactGroup.name` property because it makes real sense.

Reply:
> there is `name` property for `ArtifactGroup` class there.

I'll drop it since I don't need it. I forgot to do it.

>  anyway, I think we should have `ArtifactGroup.name` property because it makes real sense.

I don't see any sense right now since it is useless completely right now in the current implementaion.

Reply:
If you feel that it may be good to have it one day, we will add it one day. It is useless right now.

Reply:
Excuse me for joining in so late in this discussion. @fuss86: Would you mind explaining how it's used right now? I've had a look at the other pull request, but I'm not really sure I get it.